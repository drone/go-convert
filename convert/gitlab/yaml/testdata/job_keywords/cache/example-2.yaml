# https://docs.gitlab.com/ee/ci/yaml/#cachekey

cache-job:
  script:
    - echo "This job uses a cache."
  cache:
    key: binaries-cache-$CI_COMMIT_REF_SLUG
    paths:
      - binaries/