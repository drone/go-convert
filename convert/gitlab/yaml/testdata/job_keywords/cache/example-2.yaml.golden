cache-job:
    cache:
        paths:
            - binaries/
        key: binaries-cache-$CI_COMMIT_REF_SLUG
    script:
        - echo "This job uses a cache."
